### PR TITLE
fix(global): :bug: update NativeAppBar code

### PR DESCRIPTION
**Pull request by PritamBag**
## Description

Here position is written in hardcoded, which is not needed, we can pass the position in prop. So I updated this, Also one of the reasons behind the updation is this position: fixed effects in CoreSnackbar.docs.js

Ref: #121

## Related Issues

https://github.com/wrappid/guide-module/issues/252

## Testing
I use CoreAppBar in CoreSnackbar.docs.js, this works properly.

## Checklist

- [X] I have performed a thorough self-review of my code.
- [ ] I have added or updated relevant tests for my changes.
- [X] My code follows the project's style guidelines and best practices.
- [ ] I have updated the documentation if necessary.
- [ ] I have added or updated relevant comments in my code.
- [ ] I have resolved any merge conflicts of my branch.

## Maintainer Notes

- [ ] Has this change been tested in a staging environment?
- [ ] Does this change introduce any breaking changes or deprecations?

### CODE REVIEW

File: package/nativeComponents/surfaces/NativeAppBar.js [hunk 1]
```diff
@@ -8,8 +8,8 @@ export default function NativeAppBar(props) {
   const { auth } = props;
 
   return (
-    <SCAppBar {...props} position="fixed">
-      {props.children}
+    <SCAppBar {...props}>
```
Comment (anantakumarghosh) [inline package/nativeComponents/surfaces/NativeAppBar.js]: can be self closing

**Review by techoneel — APPROVED** [review]
Good to go

**Review by anantakumarghosh — DISMISSED** [review]
Please implement requested changes, feel free to ask doubts.

File: package/nativeComponents/surfaces/NativeAppBar.js (hunk 1 above)
Comment (techoneel) [reply package/nativeComponents/surfaces/NativeAppBar.js]: <SCAppBar {...props} />

File: package/nativeComponents/surfaces/NativeAppBar.js [hunk 2]
```diff
@@ -8,8 +8,8 @@ export default function NativeAppBar(props) {
   const { auth } = props;
```
Comment (techoneel) [inline package/nativeComponents/surfaces/NativeAppBar.js]: Remove this

Comment (anantakumarghosh) [reply package/nativeComponents/surfaces/NativeAppBar.js]: Thanks for mentioning!

Comment (anantakumarghosh) [reply package/nativeComponents/surfaces/NativeAppBar.js]: Also remove `// eslint-disable-next-line no-unused-vars` above the mentioned line in this comment.

**Review by techoneel — APPROVED** [review]
Good to go

**Pull request merged**